docs/versioned_docs/version-2.11.0/deprecated/releases-1x.md

```diff
@@ -0,0 +1,200 @@
+- Adding volume metric information for ZFS Local PV. These metrics can be collected using Prometheus.
+- Add support for configuring volume policies such as tolerations, nodeSelector , priorityClass, resourceLimits for main and sidecar target pod containers of cStor CSI volume using CStorVolumePolicy resource.
+- Add metrics support for cStor CSI volume which can be pulled by Prometheus to show the metrics in k8s cluster. Available metrics are `Total capacity`, `Used capacity` and `Available capacity` in Bytes.
+- Add raw block volume support for cStor CSI volume to be attached as Raw Block Volume to pods.
+- Add support for performing CSPC general validation in admission server. Some of the checks are included based on scenarios like use of duplicate block devices, duplicate nodes, block device should not be claimed by other CSPC/third party, the capacity validations of replacing block devices etc.
+- Add support for new setting requests and limits to cStor pool pod sidecar containers via poolConfig. It will take default auxResource values if it is not specified in poolConfig for a particular pool. The default can be specified in `spec.auxResources.requests` and `spec.auxResources.limits`. These values will be applied for all the pool configuration mentioned in the CSPC spec. It is also possible to specify separate auxResource values to each pool separately by adding those details to the poolConfig for a particular pool.
+- Configure Jiva CSI Driver to listen to custom ports for metrics.
+- Add metrics support for Jiva CSI volume which can be pulled by Prometheus to show the metrics in k8s cluster. Available metrics are `Total capacity`, `Used capacity` and `Available capacity` in Bytes.
+
+**Additional details:**
+
+- [Release Notes](https://github.com/openebs/openebs/releases/tag/1.6.0)
+- [Upgrade Steps](/v160/docs/next/upgrade.html)
+
+## 1.5.0 - Dec 15 2019
+
+**Change summary:**
+- Support BlockVolumeMode for OpenEBS Local PV backed by devices
+- Support ZFS as a filesystem type for OpenEBS ZFS Local PV.
+- Support for Block Device Replacement via the cStor YAML file (using new schema)
+- Support resizing and remounting of Volumes when using cStor CSI Driver
+- Support for generating of ARM builds for cStor Data Engine.
+- Introduce block device hierarchy to NDM. 4 fields `Parent` ,`Partitions`, `Holders` and `Slaves` are used in defining the hierarchy tree of a device. Also, all the dependent devices of the discovered block device will be logged during the initial udev scan to get the disk level hierarchy of each node in the cluster. 
+- Add support for applications to provision a "zfs" filesystem directly in the ZFS POOL storage which will get the optimal performance. 
+- Enhanced the cStor pools to handle auto scale down scenarios to avoid shutting down the node where cStor pool is running. This is achieved by adding cluster-autoscaler.kubernetes.io/safe-to-evict": "false"  to the annotation of each cStor pool pod.
```

Comment:
```suggestion
- Enhanced the cStor pools to handle auto scale down scenarios to avoid shutting down the node where cStor pool is running. This is achieved by adding `cluster-autoscaler.kubernetes.io/safe-to-evict": "false"`  to the annotation of each cStor pool pod.
```